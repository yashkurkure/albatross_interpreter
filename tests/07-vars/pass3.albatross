if (a+b) {
  c := d;
  return e;
  repeat (f) {
    g := 1 + a;
  }
  while (g) {
    a := c + d;
  }
  return zzz;
}
