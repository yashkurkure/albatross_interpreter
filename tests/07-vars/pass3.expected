((a)+(b))
(d)
(e)
(f)
((1)+(a))
(g)
((c)+(d))
(zzz)

